docsrc/credits_changelog.dxx

```diff
@@ -73,12 +73,34 @@
 
       <li> David St&ouml;ckel contributed the 3D convex hull functionality.
 
+      <li> lnw contributed fixes for C++ compilers in 2025.
```

Comment:
@lnw not sure if you want to include your full name. Please make suggestions

Reply:
'lukas wirz' works, but just lnw is also fine.

Reply:
```suggestion
      <li> Lukas Wirz (lnw) contributed fixes for C++ compilers in 2025.
```